web/static/internal/apps/pos/barcode-menu.js

```diff
@@ -13,7 +13,7 @@ export default {
       defaultCents: 100,
       barcode: "488348702402",
     },
-    Espresso: {
+    "Espresso": {
```

Comment:
Ideally this should be in its own PR, but it's small enough that we can just keep it in this one